cores/esp32/IPAddress.h

```diff
@@ -91,6 +91,6 @@ class IPAddress: public Printable
     friend class DNSClient;
 };
 
-const IPAddress INADDR_NONE(0, 0, 0, 0);
+const IPAddress IP_ADDR_NONE(0, 0, 0, 0);
```

Comment:
this is used in many many many esp and arduino libraries. you can not change it like that. See how that is done for other libs included (WiFiClient etc.)